packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -30,23 +29,12 @@ export class SkipLink extends GOVUKFrontendComponent {
       throw new ElementError($module, {
         componentName: 'Skip link',
         identifier: '$module',
-        expectedType: window.HTMLAnchorElement
+        expectedType: HTMLAnchorElement
       })
     }
 
     this.$module = $module
-
-    // Check for linked element
-    try {
-      const $linkedElement = this.getLinkedElement()
-      this.$linkedElement = $linkedElement
-    } catch (error) {
```

Comment:
@domoscargin It got a bit tricky to handle all the error variations in a try/catch

Now that we have `ElementError` we can throw the same error type